db/migrate/20151015205925_create_sponsor.rb

```diff
@@ -0,0 +1,11 @@
+class CreateSponsor < ActiveRecord::Migration
+  def change
+    create_table :individual_sponsors do |t|
```

Comment:
You want the table to be named the same as the class. ie the table should be called sponsors or the model be called IndividualSponsors.

Otherwise in the model fil you have to define self.table_name